scripts/helpers/build.js

```diff
@@ -93,7 +93,9 @@ module.exports = (options = {}) => {
       });
       process.on("exit", () => {
         // stop watching when the main process exits
-        subscription.unsubscribe();
+        if (typeof subscription?.unsubscribe === "function") {
```

Comment:
When I try to stop the functional tests using CTRL+C on my machine, I see an error where it says `subscription.unsubscribe is not a function`. This extra check fixes that error.